repo_checker.py

```diff
@@ -476,6 +481,8 @@ def get_optparser(self):
         parser = ReviewBot.CommandLineInterface.get_optparser(self)
 
         parser.add_option('--skip-cycle', action='store_true', help='skip cycle check')
+        parser.add_option('--force', action='store_true', help='force review even if project is not ready')
```

Comment:
I have debating adding a `--skip-checks` or somesuch for skipping the normal checks on regular run. `--force` seems a bit ambiguous especially when it only skips certain checks.

Reply:
I don't mind, maybe I missed some cases that I didn't run into. Those should be covered by --force too of course. I already spotted the "build unchanged" case now.